NewForce-Cohort-5/nutshell-nitro-menance-overboard
# Ab nmo fixes

# Description
- Just removed some commented code and removed message button from the ui
- Remove Message button from the ui
- Resolve merge conflict when pulling main
- remove friend component from the ui we may not have the time to finish the friend stretch goal
- refactor article and event components to only show edit and delete actions for the user who posted them
- sort event by upcoming date
# Testing Instructions
With servers running. Just make sure that console logs dont come from main.js as well as a message button doesnt appear in the ui of the dashboard in the messaging section. Create an event and an article and make sure another user cannot delete or edit their posts. Make sure events posted display in the order of upcoming date.
# Checklist:
